pkg/build

```diff
@@ -22,43 +22,26 @@ WEBPACK_PACKAGES = \
 	tuned \
 	users \
 	$(NULL)
```

Comment:
@KKoukiou : Wrt the "mega-glob": If you have trouble doing this with `$(wildcard)`, starter-kit and friends have done this with a simple `find` for many years without any trouble: https://github.com/cockpit-project/starter-kit/blob/main/Makefile#L83  This is a good way to start, and can be optimized later if anyone cares.